geopolars/src/geoseries.rs

```diff
@@ -162,6 +162,13 @@ pub trait GeoSeries {
     /// ```
     fn skew(&self, xs: f64, ys: f64, origin: TransformOrigin) -> Result<Series>;
 
+    /// Returns a Series containing the distance to aligned other. The operation works on a 1-to-1 row-wise manner.
+    ///
+    /// # Arguments
+    ///
+    /// * `other` - The Geoseries (elementwise) or geometric object to find the distance to.
```

Comment:
> or geometric object 

Does this work with a single geometry passed in as `other`? It looks like it needs other to be another GeoSeries

Reply:
i will add this feature

Reply:
Before supporting both, I think it would be good to first discuss and merge https://github.com/geopolars/geopolars/pull/42, so that we have a broadcasting approach that will work with all functions

Reply:
Good idea cause i was optimistic at first. It is easy to calculate against single value. But after a thought i realized that I would need to have 3 functions instead of one i think as there is no easy way for overload. So broadcasting would be nice.

Ok i will leave it as it is